src/utils/profileUtils.ts

```diff
@@ -76,6 +76,16 @@ export function hashCode(text: string) {
   return hash;
 }
 
+export function getRandomEmojiWithColor(
```

Comment:
actually does this need to be random? can it just go thru "index + 1 % length" thru the avatars list (a bit more consistent round-robin selection for a user going thru the avatar selection, also matches the logic we have in the contacts sheet

Reply:
let's do this